

Invoking dbt with ['seed', '--select', 'seed', '--full-refresh']
[0m13:21:44  Running with dbt=1.7.18
[0m13:21:44  Registered adapter: firebolt=1.6.3
[0m13:21:44  Found 12 models, 1 seed, 0 sources, 0 exposures, 0 metrics, 420 macros, 0 groups, 0 semantic models
[0m13:21:44  
[0m13:21:45  Concurrency: 1 threads (target='default')
[0m13:21:45  
[0m13:21:45  1 of 1 START seed file seed .................................................... [RUN]
[0m13:21:46  1 of 1 OK loaded seed file public.seed ......................................... [[32mCREATE 6[0m in 1.06s]
[0m13:21:46  
[0m13:21:46  Finished running 1 seed in 0 hours 0 minutes and 2.08 seconds (2.08s).
[0m13:21:46  
[0m13:21:46  [32mCompleted successfully[0m
[0m13:21:46  
[0m13:21:46  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1


Invoking dbt with ['run', '--select', 'not_found_unique_key', '--full-refresh']
[0m13:21:46  Running with dbt=1.7.18
[0m13:21:46  Registered adapter: firebolt=1.6.3
[0m13:21:46  Found 12 models, 1 seed, 0 sources, 0 exposures, 0 metrics, 420 macros, 0 groups, 0 semantic models
[0m13:21:46  
[0m13:21:47  Concurrency: 1 threads (target='default')
[0m13:21:47  
[0m13:21:47  1 of 1 START sql incremental model not_found_unique_key ........................ [RUN]
[0m13:21:48  1 of 1 OK created sql incremental model not_found_unique_key ................... [[32mSUCCESS[0m in 0.94s]
[0m13:21:48  
[0m13:21:48  Finished running 1 incremental model in 0 hours 0 minutes and 1.90 seconds (1.90s).
[0m13:21:48  
[0m13:21:48  [32mCompleted successfully[0m
[0m13:21:48  
[0m13:21:48  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1


Invoking dbt with ['run', '--select', 'not_found_unique_key']
[0m13:21:48  Running with dbt=1.7.18
[0m13:21:48  Registered adapter: firebolt=1.6.3
[0m13:21:48  Found 12 models, 1 seed, 0 sources, 0 exposures, 0 metrics, 420 macros, 0 groups, 0 semantic models
[0m13:21:48  
[0m13:21:49  Concurrency: 1 threads (target='default')
[0m13:21:49  
[0m13:21:49  1 of 1 START sql incremental model not_found_unique_key ........................ [RUN]
[0m13:21:50  1 of 1 ERROR creating sql incremental model not_found_unique_key ............... [[31mERROR[0m in 0.94s]
[0m13:21:50  
[0m13:21:50  Finished running 1 incremental model in 0 hours 0 minutes and 1.93 seconds (1.93s).
[0m13:21:50  
[0m13:21:50  [31mCompleted with 1 error and 0 warnings:[0m
[0m13:21:50  
[0m13:21:50    Runtime Error in model not_found_unique_key (models/not_found_unique_key.sql)
  Error executing query:
  {
    "errors": [
      {
        "description": "Line 3, Column 17: Column 'thisisnotacolumn' does not exist."
      }
    ],
    "query": {
      "query_id": "c023990d-2a94-42d2-90a8-a25894fb30c2",
      "request_id": "818240d8-c82a-488a-b252-30d9bdaae737"
    },
    "statistics": {
      "elapsed": 0.0
    }
  }
  
[0m13:21:50  
[0m13:21:50  Done. PASS=0 WARN=0 ERROR=1 SKIP=0 TOTAL=1
